pkg/cmd/cli/backuplocation/set.go

```diff
@@ -106,6 +112,17 @@ func (o *SetOptions) Run(c *cobra.Command, f client.Factory) error {
 	}
 
 	location.Spec.Default = o.DefaultBackupStorageLocation
+
+	for k, v := range o.Credential.Data() {
```

Comment:
Could the credential flag allow to pass/ multiple key=value, such as `key1=value1,key2=values2`?
If not, should we inform the user of the correct way to use this flag?

Reply:
I was assuming it's 1-to-1 from the conversations and design doc, but let me confirm it.

@zubron do we know yet if we want 1.1 or 1.*? I'm fine with putting this on wip if we can't make this call yet.